### PR TITLE
Add tx example

### LINKED ISSUE #9: 1 more example?
There's an example of receiving data.  If someone could find the time to contribute an example of using GenericTransmit, that would be really helpful.

### PR DESCRIPTION
**Pull request by kattni**
Fixes #9

### CODE REVIEW

File: examples/ir_transmit.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import time
+import adafruit_irremote
+import pulseio
+import board
+import digitalio
+
+# Create a button object to trigger IR transmit
+button = digitalio.DigitalInOut(board.D4)
+button.direction = digitalio.Direction.INPUT
+button.pull = digitalio.Pull.DOWN
```
Comment (caternuson) [inline examples/ir_transmit.py:11]: should this be `digitalio.Pull.UP` to match essentials guide, etc? would require changing the conditional in the loop

File: examples/ir_transmit.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# Create a 'pulseio' output, to send infrared signals on the IR transmitter @ 38KHz
+pwm = pulseio.PWMOut(board.IR_TX, frequency=38000, duty_cycle=2 ** 15)
+pulseout = pulseio.PulseOut(pwm)
+# Create an encoder that will take numbers and turn them into NEC IR pulses
+encoder = adafruit_irremote.GenericTransmit(header=[9500, 4500], one=[550, 550],
+                                            zero=[550, 1700], trail=0)
+
+while True:
+    if button.value:
```
Comment (caternuson) [inline examples/ir_transmit.py:21]: see comment above about using `digitalio.Pull.UP`

Comment (caternuson) [inline examples/ir_transmit.py:21]: add a `print("sending code")` or something here so there's indication in REPL

**Review by caternuson — CHANGES_REQUESTED** [review]
Looks good. Tested and works. Requests are basically cosmetic.

File: examples/ir_transmit.py (hunk 1 above)
Comment (kattni) [reply examples/ir_transmit.py:11]: The buttons are backwards on the CPX. So either this or the loop will be backwards for CPX or an external button. We'll leave it as CPX for now and consider changing it if we receive a bunch of feedback.

File: examples/ir_transmit.py (hunk 2 above)
Comment (kattni) [reply examples/ir_transmit.py:21]: Sounds good! Adding now.

File: examples/ir_transmit.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import time
+import adafruit_irremote
+import pulseio
+import board
+import digitalio
+
+# Create a button object to trigger IR transmit
+button = digitalio.DigitalInOut(board.D4)
+button.direction = digitalio.Direction.INPUT
+# This is for CPX. For an external button
```
Comment (caternuson) [inline examples/ir_transmit.py]: Comment seems truncated?
"Using internal pull downs for CPX button compatibility. For other external buttons, either change or do the same."

**Review by caternuson — CHANGES_REQUESTED** [review]
one more

File: examples/ir_transmit.py (hunk 3 above)
Comment (kattni) [reply examples/ir_transmit.py]: Oops. I got halfway though this idea and gave up on it, forgot to delete it. Fixing.

**Review by caternuson — APPROVED** [review]

**Pull request merged**